Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM node:10
 RUN groupadd --gid 504 jenkins \
   && useradd --uid 504 --gid jenkins --shell /bin/bash --create-home jenkins
 
-ENV YARN_VERSION 1.12.3
+ENV YARN_VERSION 1.21.1
```

Comment:
I _think_ we we need to rerun `yarn install` in case there are difference in `yarn.lock`? I thought there was a SHA or something added into the `yarn.lock` files. We also mention yarn in our `package.json` under `engines`, and in `scripts/reset-environment.sh`

Reply:
![image](https://user-images.githubusercontent.com/1915775/70826538-2e391380-1db5-11ea-8b4e-1aed9a12138a.png)

That's more like I expected! Lol

Reply:
yeah 😂- thanks for the help with this. good callouts

Reply:
i think i'm just going to leave the yarn.lock alone- i've been running on the latest yarn for months and haven't had any issues. i know we have a lot of dependencies we need to update. I don't think we need to update our yarn.lock to address the security issue in question.

Reply:
I'm afraid the next eng to run a `yarn install` is going to be the one with the giant `yarn.lock` commit. Most likely will be @kelsonic bumping the Formation version lol